src/game.js

```diff
@@ -0,0 +1,115 @@
+const _ = require('lodash')
+const Betting = require('./betting')
+const currency = require('./currency')
+
+module.exports = class Game {
+  constructor() {
+    this.minBet = 10;
+    this.maxBet = 500;
+    this.CBet = 10;
```

Comment:
I would consider more intention revealing variable names - `CBet` does not convey any meaningful clue as to its purpose (nor do `minCBet`, `midCBet`, etc. )